plugins/SnapOnMove/src/main/java/org/micromanager/plugins/snaponmove/MainController.java

```diff
@@ -387,6 +392,9 @@ private void initializeMonitoredValues() {
 
    @Subscribe
    public void onZMoved(StagePositionChangedEvent e) {
+      if (pausedForAcquisition_) {
```

Comment:
Best to consistently synchronize on `pauseLock_` when accessing `pausedForAcquisition_`.